x-pack/plugins/lens/public/datasources/form_based/operations/definitions/metrics.tsx

```diff
@@ -52,6 +54,13 @@ function isTimeSeriesCompatible(type: string, timeSeriesMetric?: string) {
   return timeSeriesMetric !== 'counter' || ['min', 'max'].includes(type);
 }
 
+function timeScaleToUnit(t?: string) {
+  if (!t || t === 's') return 'second';
+  else if (t === 'h') return 'hour';
+  else if (t === 'm') return 'minute';
+  else return 'day';
```

Comment:
nit: no need for `else`
```suggestion
 if (t === 'h') return 'hour';
 if (t === 'm') return 'minute';
 return 'day';
```